docs/design/webtransport.md

```diff
@@ -22,7 +22,7 @@ JavaScript SDK creates a WebTransport with a QUIC agent when QUIC agent is enabl
 
 ## Limitations
 
-WebTransport only supported in Chrome since 87. It's not enabled by default. To enable it, you need to [register the origin trail](https://web.dev/webtransport/#register-for-ot) or start Chrome with flag `--enable-experimental-web-platform-features`.
+WebTransport is only supported in Chrome since 97. Custom certificate is not enabled by default. It's controlled by "WebTransportCustomCertificates", see [this change](https://chromium-review.googlesource.com/c/chromium/src/+/3208466) for detailed information.
```

Comment:
supported by Chrome since version 97. Certificate that is self-signed or non-authorized CA signed is not enabled by default.

Reply:
Custom certificate requirement is defined in WebTransport spec. I added a link and some words to explicitly indicate it includes self-signed and untrusted CA signed certificates.